nlplot/nlplot.py

```diff
@@ -17,7 +18,8 @@
 
 import gensim
 import pyLDAvis.gensim
-pyLDAvis.enable_notebook()
+if is_notebook():
+    pyLDAvis.enable_notebook()
```

Comment:
#4
jupyter notebooks and the python interpreter, a mechanism to work with both.